runtime/constants.go

```diff
@@ -51,6 +51,11 @@ const (
 	clientHTTPUnmarshalError = "client.http-unmarshal-error"
 	// clientTchannelReadError is the metric for tracking errors in reading tchannel response
 	clientTchannelUnmarshalError = "client.tchannel-unmarshal-error"
+
+	// shadow headers and environment
+	shadowHeader      = "X-Uber-Shadow-Request"
```

Comment:
Open source code cannot have uber specific anything. You should probably have a generic shadow header mechanism and set it to a specific value in the uber internal use-case

Reply:
Valid point. I have renamed it to a more generic shadow header "X-Shadow-Request"

Reply:
done